cg/store/models.py

```diff
@@ -829,6 +826,11 @@ class Sample(Base, PriorityMixin):
     def __str__(self) -> str:
         return f"{self.internal_id} ({self.name})"
 
+    @property
+    def flow_cells(self) -> list[Flowcell]:
```

Comment:
Should this be `flowcells` for consistency or should we keep moving towards flow_cells?

Reply:
I think flow_cell is good

Reply:
I relate a `flow_cell` with a `FlowCellDirectoryData` and a `flowcell` with a `Flowcell`. Maybe we should keep the old way for now and in the future consider a renaming. There are still many instances in the code using `flowcell` so this could be part of the scope of another PR

Reply:
@diitaz93 I think your association is logical given our code, but in reality flowcell is just a typo (swedifying an english word). I think I will go for flow_cell